deploy/huey.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+huey_consumer.py portality.tasks.consumer.main_queue
```

Comment:
I wonder if we can reuse this across environments, so in production sidekiq will start this script, which will start huey. That way we all start huey the same way. I can foresee there might be trouble with that .. but we'll see.

For now this is fine.